spec/spec_helper.rb

```diff
@@ -30,6 +30,7 @@ def errback
   config.order = 'random'
   config.include SlangerHelperMethods
   config.fail_fast = true
+  config.expect_with(:rspec) { |c| c.syntax = [:should, :expect] }
```

Comment:
I've run transpec on it, let me know if you still want me to split it to a new PR